Lib/test/test_string/test_templatelib.py

```diff
@@ -148,6 +148,15 @@ def test_iter(self):
         self.assertEqual(res[1].format_spec, '')
         self.assertEqual(res[2], ' yz')
 
+    def test_exhausted(self):
+        # gh-134119
```

Comment:
```suggestion
        # See https://github.com/python/cpython/issues/134119.
```

With an URL, I can just click it from my IDE and I find it useful